corgi/tasks/management/commands/loadcomposedata.py

```diff
@@ -60,6 +70,6 @@ def handle(self, *args, **options) -> None:
         for build_id in build_ids:
             build_id = int(build_id)
             if options["inline"]:
-                slow_fetch_brew_build(build_id)
+                slow_fetch_brew_build(build_id, force_process=force_process)
```

Comment:
You can just pass `options["force"]` here (and below), it'll already be True or False based on the input.